package.json

```
@@ -90,7 +90,9 @@
   },
   "lint-staged": {
     "*.{js,css,md,json}": [
-      "prettier --write",
+      "prettier --write"
+    ],
+    "*.{js,md,json}": [
```

Not sure why was this changed?

```suggestion
      "prettier --write",
```

---

esling goes through css files? Also the ss isn't working